docs/plugins/purchase.md

```diff
@@ -36,22 +38,16 @@ Purchase.buy(assetId).then(() => {
 })
 ```
 
-In case the CSP back-end does _not_ completely follow the expected blueprint or can not be configured properly via `setup`, the steps in the `buy` method can also be implemented manually, via the other methods exposed by the Purchase plugin (`signature`, `payment`, `subscribe` and `confirm`).
-
-
-### Setup
-
-The `setup`-method is used to configure the Purchase plugin to match with a specific CSP backend setup used for providing available assets and storing purchased asset records.
+### setup
 
-The `setup`-method should be called once, before any other method on the Purchase plugin is called.
+You use the `setup` method to configure the Purchase plugin for matching with a specific CSP backend setup, which is used for providing available assets and storing purchased asset records.
 
-The method accepts a config `object` as its only argument. This object can have 2 keys: `cspUrl` and `endpoints`.
+> The `setup` method should only be called once, before any other Purchase plugin method is called.
 
-`cspUrl` specifies the _base Url_ of the backend of the CSP.
+The `setup` method accepts a *configuration object* as its only argument. The object can have *2 keys*:
 
-`endpoints` can be used to customize those API endpoints that differ from the default specification (as outlined below). For each _endpoint_ that deviates from the default, specify the _uri_ (which will be appended to the `cspUrl`, unless specified as a fully qualified domain) and the REST _method_ to use.
-
-If needed you can also specify an object with custom `headers` that will be merged in with the defaults headers (`{ Accept: 'application/json', 'Content-Type': 'application/json' }`)
```

Comment:
I think this sentence was maybe added at a later stage. We need to add this info as well.

Reply:
Sentence about custom `headers` added.